tests/components/senseme/test_config_flow.py

```diff
@@ -88,6 +88,7 @@ async def test_form_user_manual_entry(hass: HomeAssistant) -> None:
         )
         await hass.async_block_till_done()
 
+    assert result3["type"] == RESULT_TYPE_CREATE_ENTRY
```

Comment:
Please assert the result type on line 135 too.

Reply:
👍  done